capstone/src/main/java/com/google/univiz/api/CollegeId.java

```diff
@@ -0,0 +1,4 @@
+package com.google.univiz.api;
+
+/** */
```

Comment:
let's avoid empty javadoc, if possible...

Reply:
done! it's very simple, but I think it works for right now